frontend/simple/views/containers/proposals-form/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Form.vue'
```

Comment:
I'm not super thrilled with these `index.js` files (see one of the comments above for details) please consider some simpler, less elaborate form of organization

Reply:
Hm... I removed, but we need to define well how to use imports. Because in some cases this `index.js` still makes sense, specially when it can export multiple modules.